phalcon/Environment/Adapter/PhpDotEnv.zep

```diff
@@ -0,0 +1,60 @@
+namespace Phalcon\Environment\Adapter;
+
+use DotEnv\DotEnv;
```

Comment:
I'm wondering if this dependency could cause some issues. If you want to use this adapter you need to have composer installed, vlucas/phpdotenv installed and setup the autoloader to use the vendor dir.
I can imagine use cases (for example cronjobs) where you would like to use this service but prefer not to have composer etc configured. 
Would it be possible to replace DotEnv with our own implementation or is this too much work?

Reply:
I think the task is to big too rewrite this in Phalcon now but would love to see this in the feature so we don't depend on the package and only extend on php extensions.
Should we add this as recommended to the composer.json?

Reply:
@ruudboon Phalcon uses composer.json only for testing purposes. Usually users installs `phalcon.so` and don't use Phalcon's `composer.json`.